### PR TITLE
remove --release from regtest README

### PR DESCRIPTION
**Pull request by zancas**

### CODE REVIEW

File: regtest/README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ## Regtest Mode
 WARNING Experimental!
-The CLI can work in regtest mode, by locally running a `zcashd` and `lightwalletd`.
+The CLI can work in regtest mode, by running a local `lightwalletd` that proxies to a local `zcashd` in "regtest mode".
```
Comment (dannasessha) [inline regtest/README.md:3]: Proxies is a plural noun, so this sentence isn't quite right yet,

File: cli/src/regtest.rs [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+    // wait 5 seconds for lwd to fire up
+    // very generous, plan to tune down
+    let five_seconds = time::Duration::from_millis(5_000);
+    thread::sleep(five_seconds);
+
+    // this process does not shut down when rust client shuts down!
+    // TODO Needs a cleanup function, or something.
+    println!("lwd start section completed, lightwalletd should be running!");
+    println!("Standby, Zingo-cli should be running in regtest mode momentarily...");
+}
```
Comment (dannasessha) [inline cli/src/regtest.rs:241]: Ahw, I was hoping to make the mod.

But this all looks fine.

**Review by dannasessha — COMMENTED** [review]
Comments attached, should be ready with another nudge.

File: regtest/README.md (hunk 1 above)
Comment (zancas) [reply regtest/README.md:3]: I meant this:  https://www.wordhippo.com/what-is/the-verb-for/proxy.html

**Comment by zancas** [comment]
@AloeareV I think this is ready to merge.

**Review by AloeareV — APPROVED** [review]
Looks good! Needs a rebase before I can merge it, though!

**Comment by dannasessha** [comment]
commit [ddcda4b](https://github.com/zingolabs/zingolib/pull/48/commits/ddcda4b0f8811df6025b9e720e38f64a8ecffb16) has a change that didn't make it into #49 - but mostly this PR was covered by that PR (#49). Closing.

**Pull request closed without merging**